tests/scripts/build-release-downstream.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+set -ex
+
+# Load dot env file if available
+if [ -f .env ]; then
+    # shellcheck disable=SC2046
+    export $(grep -v '^#' .env | xargs -d '\n')
+fi
+
+MAKE='make --debug=v --output-sync'
+$MAKE IMAGES="ceph" build
```

Comment:
1. We don't need the IMAGES env var anymore. We always only build the ceph image now.
2. If you set BUILD_REGISTRY=local, it will result in an image name of something like: `local/ceph-amd64:latest`. Then you don't need to grep for the image, you can just use that directly (see separate comment)
```suggestion
$MAKE build BUILD_REGISTRY=local
```